src/Load.cpp

```diff
@@ -84,44 +84,53 @@ namespace ale {
                            "This Usually indicates an error in the Ale Python Library."
                            "Check if Installed correctly and the function ale.loads exists.");
      }
+      
 
      // Create a Python tuple to hold the arguments to the method.
      PyObject *pArgs = PyTuple_New(3);
      if(!pArgs) {
        throw runtime_error(getPyTraceback());
      }
-
+      
      // Set the Python int as the first and second arguments to the method.
      PyObject *pStringFileName = PyUnicode_FromString(filename.c_str());
      PyTuple_SetItem(pArgs, 0, pStringFileName);
+     Py_INCREF(pStringFileName); // take ownership of reference
 
      PyObject *pStringProps = PyUnicode_FromString(props.c_str());
      PyTuple_SetItem(pArgs, 1, pStringProps);
-
+     Py_INCREF(pStringProps); // take ownership of reference
+     
      PyObject *pStringFormatter = PyUnicode_FromString(formatter.c_str());
      PyTuple_SetItem(pArgs, 2, pStringFormatter);
+     Py_INCREF(pStringFormatter); // take ownership of reference 
```

Comment:
Once the strings are added to the tuple, the tuple takes ownership of the reference, so we need to increment the reference to explicitly maintain ownership.